src/Kdyby/Events/DI/EventsExtension.php

```diff
@@ -94,7 +94,7 @@ public function loadConfiguration()
 
 		Nette\Utils\Validators::assertField($config, 'subscribers', 'array');
 		foreach ($config['subscribers'] as $subscriber) {
-			$def = $builder->addDefinition($this->prefix('subscriber.' . md5(Nette\Utils\Json::encode($subscriber))));
+			$def = $builder->addDefinition($this->prefix('subscriber.' . md5(Nette\Utils\Json::encode((array) $subscriber))));
```

Comment:
Nice catch - this should be probably fixed the same way [as the exact same issue in kdyby/console](https://github.com/Kdyby/Console/commit/0fb546c3a7fbd48feed8bd7275316fd5e7f344c9)